variable/include/scipp/variable/string.h

```diff
@@ -30,9 +30,10 @@ to_string(const std::pair<Dim, VariableConstView> &coord);
 SCIPP_VARIABLE_EXPORT std::string
 to_string(const std::pair<std::string, VariableConstView> &attr);
 
-SCIPP_VARIABLE_EXPORT std::string
-format_variable(const std::string &key, const VariableConstView &variable,
-                const Dimensions &datasetDims = Dimensions());
+SCIPP_VARIABLE_EXPORT std::string format_variable(
+    const std::string &key, const VariableConstView &variable,
+    std::optional<std::reference_wrapper<const Dimensions>> datasetDims =
+        std::nullopt);
```

Comment:
How about `std::optional<Dimensions>`?

Reply:
Could do. I used the reference wrapper because it was a ref before. Do we care about making copies here? (Have two copies,one in `format_variable` and one in `make_dim_labels`.

Reply:
`Dimensions` is just 64 Byte, I'd say nicer code is more importent here.